lab_5_scrapper/scrapper.py

```diff
@@ -120,13 +241,20 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(full_url, article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        main_bs = article_soup.find('div', {'class': 'item'})
+        content = main_bs.findAll('p')
+        if content:
+            for paragraph in content:
+                self.article.text += paragraph.text
```

Comment:
do the paragraphs always contain space as the last symbol? if not, it looks like you're joining the last word of the previous paragraph with the first word of the next one

Reply:
The text was displayed with two lines in between when I tested